Chiazokam/iReporterApp
# #162367762 create api endpoint to delete intervention

#### What does this PR do?
- implements tests for the delete intervention api endpoint
- implements the api endpoint
- changes the route and the return message of the root route
- seperates routes into a router folder and import to server

#### Description of Task to be completed?
- write tests for the delete intervention api endpoint
- implement the api endpoint
- change the route and the return message of the root route
- seperate routes into a router folder and import to server

#### How should this be manually tested?
- make sure to have node installed on ur pc
- cd into the desired destination
- clone repo into the folder
- cd into the iReporterApp folder
- run
  npm run test
- the test results are then seen

#### What are the relevant pivotal tracker stories?
- [#162367762](https://www.pivotaltracker.com/story/show/162367762)

#### Screenshots (if appropriate)
Below is a sample of a returned data
![image](https://user-images.githubusercontent.com/26940294/49644146-ee0fba00-fa17-11e8-847b-ef17f7061298.png)
